content/help/docs/deposit/about-records/contents.lr

```diff
@@ -20,13 +20,13 @@ A **record** consists of:
 - **Files:** The actual data for the digital research object.
 - **Persistent identifier**: A globally unique persistent identifier (in our case a [Digital Object Identifier](https://www.doi.org)) used for identification of the record.
 
-#### Metadata
+##### Metadata
```

Comment:
I kind of like the bigger headers before 😅 

Is it changing something practically if we keep them as `h4`?